tests/test_sklearn_permutation_importance.py

```diff
@@ -13,7 +13,6 @@
 from eli5.sklearn import PermutationImportance
 from .utils import format_as_all
 
-
```

Comment:
according to pep8, function should be separated by two lines, not but one - could you please keep the line here, and add a line before the new test function?